docker-compose/is/dockerfiles/is/Dockerfile

```diff
@@ -21,7 +21,7 @@ FROM docker.wso2.com/wso2is:5.11.0.0
 LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>"
 
 # build arguments for external artifacts
-ARG MYSQL_CONNECTOR_VERSION=8.0.29
+ARG MYSQL_CONNECTOR_VERSION=8.4.0
 
 # add MySQL JDBC connector to server home as a third party library
-ADD --chown=wso2carbon:wso2 https://repo1.maven.org/maven2/mysql/mysql-connector-java/${MYSQL_CONNECTOR_VERSION}/mysql-connector-java-${MYSQL_CONNECTOR_VERSION}.jar ${WSO2_SERVER_HOME}/repository/components/dropins/
+ADD --chown=wso2carbon:wso2 https://repo1.maven.org/maven2/com/mysql/mysql-connector-j/${MYSQL_CONNECTOR_VERSION}/mysql-connector-j-${MYSQL_CONNECTOR_VERSION}.jar ${WSO2_SERVER_HOME}/repository/components/dropins/
```

Comment:
We should ideally remove this. Since we are doing that with Ziyam's effort, let's go ahead and bump the version for this update.